micromagneticdata/drive.py

```diff
@@ -417,7 +415,7 @@ def slider(self, description="step", **kwargs):
             value=0, min=0, max=self.n - 1, step=1, description=description, **kwargs
         )
 
-    def to_xarray(self):
+    def to_xarray(self, *args, **kwargs):
```

Comment:
Please explain that `args` and `kwargs` are forwarded to `Field.to_xarray`.

Reply:
I have updated the docstrings. Have a look.

Reply:
Looks good. I've added a few minor comments.